### PR TITLE
Fix adding reset user without PUK to a team

### PR DESCRIPTION
**Pull request by patrickxb**

### CODE REVIEW

File: go/teams/teams.go [hunk 1]
```diff
@@ -412,17 +412,17 @@ func (t *Team) InviteEmailMember(ctx context.Context, email string, role keybase
...
 }
 
-func (t *Team) inviteKeybaseMember(ctx context.Context, uid keybase1.UID, role keybase1.TeamRole, resolvedUsername libkb.NormalizedUsername) (keybase1.TeamAddMemberResult, error) {
-	t.G().Log.Debug("team %s invite keybase member %s", t.Name(), uid)
+func (t *Team) inviteKeybaseMember(ctx context.Context, uv keybase1.UserVersion, role keybase1.TeamRole, resolvedUsername libkb.NormalizedUsername) (keybase1.TeamAddMemberResult, error) {
+	t.G().Log.Debug("team %s invite keybase member %s", t.Name(), uv)
 	invite := SCTeamInvite{
 		Type: "keybase",
-		Name: uid.String(),
+		Name: uv.PercentForm(),
```
Comment (patrickxb) [inline go/teams/teams.go]: this is the fix

**Comment by maxtaco** [comment]
Uh oh, I think tests are failing legitimately now --- or is it waiting for a server update?

**Comment by patrickxb** [comment]
server update

2017-08-03 19:06:21.454259807 +0000 UTC api.go:916: [D] - API call sig/multi error: bad keybase name given: 73a5bcf43d08ab1385fa3958aa354819%0 (error 2639)

**Comment by maxtaco** [comment]
cool hopefully the rerun will work

**Comment by patrickxb** [comment]
still waiting on new server...started another build

**Pull request merged**